client/state/sites/selectors.js

```diff
@@ -166,3 +166,34 @@ export function isCurrentSitePlan( state, siteId, planProductId ) {
 
 	return sitePlan.product_id === planProductId;
 }
+/**
+ * Get primary site
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Object}         Site
+ **/
+export function getPrimarySite( state ) {
+	// need to figure out best way to retrieve primary site
+	// since this attribute doesn't exist by default
+	return filter( state.sites.items, ( site ) => site.primary ).shift();
```

Comment:
- I dont see 'primary' attribute on any of my sites.
- Is there a particular reason you used shift instead of `[ 0 ]` ?